src/libraries/System.IO.Pipes/src/Resources/Strings.resx

```diff
@@ -243,6 +243,9 @@
   <data name="PlatformNotSupported_MessageTransmissionMode" xml:space="preserve">
     <value>Message transmission mode is not supported on this platform.</value>
   </data>
+  <data name="PlatformNotSupported_PipeOptions_FirstPipeInstance" xml:space="preserve">
+    <value>First Pipe Instance option is not supported on this platform.</value>
+  </data>
```

Comment:
I think it'd be better to have a single message for unsupported options, e.g.
```xml
  <data name="PlatformNotSupported_UnsupportedOption" xml:space="preserve">
    <value>'{0}' is not supported on this platform.</value>
  </data>
```
and then in the code do something like:
```C#
#pragma warning disable CA1416
            if ((options & PipeOptions.FirstPipeInstance) != 0)
#pragma warning restore CA1416
            {
                throw new PlatformNotSupportedException(SR.Format(SR.PlatformNotSupported_UnsupportedOption, $"{nameof(PipeOptions)}.{nameof(PipeOptions.FirstPipeInstance)}"));
            }
```

Reply:
I agree with @stephentoub that it would be a better solution.